study.md

```diff
@@ -24,16 +24,15 @@ In your own words, define what a SPA is. In your answer, be sure to cite any
 relevant sources you consulted in your search.
 
 ```md
-<!-- your answer here -->
-```
+A single page application is a website that is one page.  Instead of hyperlinking to other pages, it links to other spots in the page -- usually to hash locations lower in the page.
```

Comment:
It can render different templates that don't require a full refresh of the page.